### DISCUSSION TITLE
Custom Registration and login page and custom role management

### DISCUSSION THREAD

**Discussion by anoop-mathew** [Budibase/budibase#6756] [General]
Is their any option to have custom Registration and login page for the user, for example i am planning to build a appointment portal using budibase and for that i need to make user who are going to book the appointment to login/registration from the from end. And when they are login in they will able to see only the options specific for that user alone ( with the help of the ur I'd)

**Comment by melohagan** [comment]
Hi @anoop-mathew 

I don't think there's a way to have custom registration/login pages, however you can customise your screens to show for users of a specific role. 

In addition, there is a concept of the `Current User` binding and relationships that allow you to get data related to only that user. 

It may be worth having a read through these links:
- https://docs.budibase.com/docs/users-table
- https://docs.budibase.com/docs/screens#access-level
- https://docs.budibase.com/docs/user-roles
